pom.xml

```diff
@@ -88,8 +88,13 @@
 		<dependency>
 			<groupId>org.springframework.integration</groupId>
 			<artifactId>spring-integration-xml</artifactId>
+			<exclusions>
+				<exclusion>
+					<groupId>commons-logging</groupId>
+					<artifactId>commons-logging</artifactId>
+				</exclusion>
```

Comment:
Will merge as is, and let's see if we need to do anything else for SLF4J. Not sure that we do.